BricklinkSharp.Client/BricklinkApiRequestTypes.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BricklinkSharp.Client
+{
+    public enum BricklinkApiRequestTypes
```

Comment:
Naming suggestion: BricklinkApiResource?

Reply:
BricklinkApiResourceTypes?

Reply:
Singular (BricklinkApiResourceType)? You'll never have more than one at a time.